web/src/components/ResourceCard/ResourceCardRuns.tsx

```diff
@@ -27,7 +28,7 @@ const ResourceCardRuns = ({children, hasMoreRuns, hasRuns, isCollapsed, isLoadin
 
       {hasMoreRuns && (
         <S.FooterContainer>
-          <S.Link data-cy="test-details-link" onClick={onViewAll}>
+          <S.Link data-cy="test-details-link" onClick={onViewAll} href={`/test/${resourceId}`}>
```

Comment:
is `/test` correct here? Are we using this component for transactions?

Reply:
Ups you are right